components/Tour/StageMapLine.tsx

```diff
@@ -0,0 +1,81 @@
+import { Location } from "@/model/model";
+import MapboxGL from "@rnmapbox/maps";
+import { lineString, point, featureCollection } from "@turf/helpers";
+import { Layout, useTheme } from "@ui-kitten/components";
+import React from "react";
+import { Feature, FeatureCollection, LineString, Point } from "geojson";
+
+const StageMapLine = ({
+  locations,
+  stageID,
+  active,
+}: {
+  locations: Location[];
+  stageId: number;
+}) => {
```

Comment:
```suggestion
type MapLineType = {
locations: Location[];
  stageId: number;
};
const StageMapLine = ({
  locations,
  stageID,
  active,
}: MapLineType) => {
```
Oder so ähnlich, halt den Typen rausziehen